Automattic/wp-calypso
# Create Jetpack Cloud Upsell

#### Changes proposed in this Pull Request

* Add backup upsell if rewind state is "unavailable" due to a "missing_plan"

#### Desktop
<img width="300" src="https://user-images.githubusercontent.com/2810519/79018147-ee4a2900-7b27-11ea-9648-46f79293c399.png" >

#### Mobile
<img width="300" src="https://user-images.githubusercontent.com/2810519/79018159-f86c2780-7b27-11ea-82d7-9393687c6f81.png" >


#### Testing instructions

Navigate to `/backups` with a site with a Free Jetpack Plan ( that is not Jurassic.Ninja or Pressable ) backup
OR
Navigate to `/backups` and dispatch the following action via Redux Dev Tools:
```
{
  type: 'REWIND_STATE_UPDATE',
  siteId: 173211884,
  data: {
    state: 'unavailable',
    reason: 'missing_plan'
  }
}
```


Things to test:
- "Upgrade" link should go to WordPress.com/plans for the site
